core/tests/performance_framework/selenium_driver.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Contains a utility for fetching performance data using Selenium and
+Browsermob-proxy.
+"""
+
+import os
+import time
+import urlparse
+
+# We need to import browsermob-proxy and selenium. To do so, we need to add
+# add their directories to path.
+CURR_DIR = os.path.abspath(os.getcwd())
+OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
+
+DIRS_TO_ADD_TO_SYS_PATH = [
+    os.path.join(OPPIA_TOOLS_DIR, 'browsermob-proxy-0.7.1'),
+    os.path.join(OPPIA_TOOLS_DIR, 'selenium-2.53.2'),
+    os.path.join(OPPIA_TOOLS_DIR, 'xvfbwrapper-0.2.8')
+]
+
+# pylint: disable=import-error, wrong-import-position
+from browsermobproxy import Server
+from selenium import webdriver
+
+# Used for headless testing i.e, making browser windows invisible to the user.
+from xvfbwrapper import Xvfb
+# pylint: enable=import-error, wrong-import-position
+
+CHROME_DRIVER_PATH = os.path.join(
+    '..', 'node_modules', 'protractor', 'selenium', 'chromedriver_2.21')
+
+BROWSERMOB_PROXY_PATH = os.path.join(
+    '..', 'oppia_tools', 'browsermob-proxy-2.1.1', 'bin', 'browsermob-proxy')
+
+# Duration to wait for the complete page to load, otherwise XHR requests made
+# after page load will not be recorded.
+WAIT_DURATION = 0
```

Comment:
Should this be 0? Also, is this more appropriate to be a constant within ``SeleniumPerformanceDataFetcher``?